build.gradle

```diff
@@ -55,6 +55,8 @@ dependencies {
   implementation 'commons-io:commons-io:2.11.0'
   implementation "commons-beanutils:commons-beanutils:1.9.4"
   implementation "org.apache.commons:commons-lang3:3.12.0"
+  implementation 'org.json:json:20220924'
+  implementation 'javax.xml.bind:jaxb-api:2.4.0-b180830.0359'
```

Comment:
This is a beta build, we should use `2.3.1` instead.

Reply:
It is removed together with org.json